webview_panels/src/lib/altimate/altimate-components.d.ts

```
@@ -1,594 +1,490 @@
+declare interface Props extends ButtonHTMLAttributes<HTMLButtonElement> {
+    color?: string;
+}
+
+declare interface Props_10 {
+    taskLabel: keyof typeof TaskLabels;
+    context?: Record<string, unknown>;
+    onClose: () => void;
+}
+
+declare interface Props_11 {
+}
```

Biome

[error] An empty interface is equivalent to {}.

Safe fix: Use a type alias instead.


(lint/suspicious/noEmptyInterface)